nextflow_schema.json

```diff
@@ -5,6 +5,83 @@
     "description": "IRIDA Next Example Pipeline",
     "type": "object",
     "definitions": {
+        "arborview": {
+            "title": "ArborView",
+            "type": "object",
+            "description": "",
+            "default": "",
+            "properties": {
+                "av_html": {
+                    "type": "string",
+                    "default": "./assets/ArborView.html",
+                    "hidden": true
+                }
+            }
+        },
+        "gas_cluster": {
+            "title": "GAS Cluster",
+            "type": "object",
+            "description": "",
+            "default": "",
+            "properties": {
+                "gm_thresholds": {
+                    "type": "string",
+                    "default": "10,5,0"
+                },
+                "gm_delimiter": {
+                    "type": "string",
+                    "default": "\\'.\\"
+                }
```

Comment:
I think these and other inputs in this file that the user will see should have `description`, `pattern`, and maybe eventually `fa_icon` (although low priority on that one) so that they're more understandable in Irida Next. I think the last time I saw the UI for Irida Next is does show the `gm_thresholds` part, so the description needs to do a lot and stand on its own.

Reply:
I can update those in the schema! Ill take care of that after the including the test data.